### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by david-lafontant**
Requirement:
- Follow the design.
- Interaction.
- Testing.
- Technical requirements

   * The project is a single-page application (SPA) built with React and Redux.
   * The data retrieved from the API should be stored in the Redux store.
   * The data retrieved from the API  can be filtered using a Filter stateless component.
   * Every page (the main page and the pages for each item) have a unique route within the SPA.
   * The project should be deployed and accessible online (add a link to the online version in the README file).

- Documentation and video presentation

For more details about the requirement please visit this [link](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md)

### CODE REVIEW

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi  @david-lafontant 
Good job so far! You are almost done :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good job with the git-flow :+1:
- Good PR description :clap: 

## Required Changes ♻️ 
- [ ] _Check the comments below :arrow_double_down: _

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,3 +1,125 @@
...
+    - [`npm run build` fails to minify](#npm-run-build-fails-to-minify)
+
+![screenshot](Animation.gif)
+
+## Live Demo
+
+[Netlify](https://the-zoo-animal.netlify.app/)
+
+## Presentation video
+[Loom](https://www.loom.com/share/999987edd28b47f791da9dba7149a5f7)
```
[the remark this answers is not in the document]
Comment (david-lafontant) [reply README.md]: @GzimAsani very sory for this. Asked change implemented.

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,25 @@
-import logo from './logo.svg';
```
[the remark this answers is not in the document]
Comment (david-lafontant) [reply src/App.js:1]: Asked change implemented

**Review by BrianSammit — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @david-lafontant , 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it. :shipit: 

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch  :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**